includes/class-wc-stripe-payment-tokens.php

```diff
@@ -388,12 +388,13 @@ public function get_account_saved_payment_methods_list_item( $item, $payment_tok
 			$item['method']['brand'] = esc_html__( 'SEPA IBAN', 'woocommerce-gateway-stripe' );
 		}
 
-		if ( 'cashapp' === strtolower( $payment_token->get_type() ) ) {
-			/**
-			 * WC's wc_get_credit_card_type_label() function will automatically replace underscores and dashes with spaces.
-			 * Cashtags can include `_` and `-` characters and so to keep the cashtag intact, we need to avoid that by using their HTML entity.
-			 */
-			$item['method']['brand'] = str_replace( [ '_', '-' ], [ '&#95;', '&#8211' ], $payment_token->get_display_name() );
+		switch ( strtolower( $payment_token->get_type() ) ) {
```

Comment:
I think we could include the `sepa` condition inside the `switch` clause.

Reply:
👍 100% agree. I missed that.